Add possiblity to add resources before the sources build phase (#1351)
yonaskolb/XcodeGen: CHANGELOG.md

### Added

- Added support for shared breakpoints #177 @alexruperez @myihsan

> - Added support for putting the resources build phase before the sources build phase to add the #1351 @mat1th 

Reviewer: I think maybe this sentence got cutoff?
```suggestion
- Added support for `putResourcesBeforeSourcesBuildPhase` in a target #1351 @mat1th 
```
